latex/epoch.tex

```diff
@@ -409,66 +302,29 @@ \subsection{Rewards Distribution}
         \floor*{(\hat{f} - c)\cdot(1-m)\cdot\frac{t}{\sigma}} &
         \text{otherwise.}
       \end{cases} \\
-    & ~~~\where (c, m, \_) = \fun{poolCosts}~pool \\
+    & ~~~\where \\
+    & ~~~~~~~c = \fun{poolCost}~pool \\
+    & ~~~~~~~m = \fun{poolMargin}~pool \\
   \end{align*}
 
   \caption{Functions used in the Reward Splitting}
   \label{fig:functions:reward-splitting}
 \end{figure}
 
 
-Finally, the reward calculation is presented in
-Figure~\ref{fig:functions:reward-calc}. The calculation is done pool-by-pool.
-The map $\fun{rewardOnePool}$ calculates the rewards given out to each member
-of a given pool (with leader identified by
-the stake key $\var{poolHK}$). The active stake associated to each
-pool member is passed to $\fun{rewardOnePool}$
-via the $\var{actgr}$ variable, which is a subset of hash keys in this pool
-paired with the stake associated with the keys.
-
-This $\fun{rewardOnePool}$ map outputs a finite map and an $\var{avg}$
-value. The finite map is indexed
-by reward addresses. In particular, the subset of reward addresses corresponding
-to registered keys delegating to the pool with leader $\var{poolHK}$.
-A reward address of key $hk$ is paired with the individual reward value
-produced by the $\fun{indivReward}$ map applied to relevant data passed to
-$\fun{rewardOnePool}$ as well as the results of the $\fun{maxPool}$
-and $\fun{poolReward}$ calculations. The $\var{avg}$ value for the stake pool
-(i.e. the pool's ranking) is also calculated by the $\fun{poolReward}$ function.
-
-The $\fun{reward}$ calculation itself uses the $\fun{rewardOnePool}$ function to
-reward each pool's members and calculate the pool's moving averages. In order to
-do this, it is given the following parameters:
-
+Finally, the full reward calculation is presented in Figure~\ref{fig:functions:reward-calc}.
+The calculation is done pool-by-pool.
 \begin{itemize}
-\item The number of blocks made by each key this epoch
-\item The protocol parameters
-\item The total coin value of rewards to be distributed this epoch
-\item The full delegation state
-\item The set of outputs in the UTxO on the ledger
+  \item The $\fun{rewardOnePool}$ function calculates the rewards given out to each member of a
+    given pool. The pool leader identified by the stake key of the pool operator.  The function
+    returns both the rewards and the total amount of unrealized potential rewards.  The unrealized
+    amount will go to the treasury. The rewards include that of the the pool operator, provided
+    the reward account from the pool registration certificate is still registered.  If the pool
+    reward account is not registered, the operator reward is added to the unrealized total.
```

Comment:
I do wonder whether it would be useful to identify "leader" with "operator" in one place and use it consistently elsewhere?